webvtt.go

```diff
@@ -193,7 +194,9 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 			case webvttBlockNameStyle:
 				// TODO Do something with the style
 			case webvttBlockNameText:
-				item.Lines = append(item.Lines, Line{Items: []LineItem{{Text: line}}})
+				// Voice Tag to extract Speaker Name
+				voiceName, text := extractVoiceNameAndText(line)
+				item.Lines = append(item.Lines, Line{Items: []LineItem{{Text: text}}, VoiceName: voiceName})
```

Comment:
I'd rather create a method that returns a `Line` directly, that way you can call directly `item.Lines = append(item.Lines, theFunction(line))`

Reply:
done.